src/components/StatesButton/StatesButton.st.css

```diff
@@ -5,7 +5,6 @@
 
 .root {
     -st-extends: TPAButton;
```

Comment:
We also talked about adding a 
```css
-st-mixin: TPAButton(
  /* All the overridable variables */
)
```
right?

Reply:
I think it's out of this specific PR's scope.. I'll quick check if this can be done easily or it supposed to be a separate feature

Reply:
I thought we wanted to add the underline text-decoration, no?

Reply:
Taking the text out of the div solves it. No need for extra CSS.

Reply:
I see.
But we still can't merge though, right?

Reply:
Yes, I'll push and update in the next few days

Reply:
💪